src/Compilers/Core/Portable/InternalUtilities/RoslynExperiments.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis;
+
+/// <summary>
+/// IDs of currently active experimental APIs.
```

Comment:
Honestly, we should probably keep these ID's around, even after they ship, as we generally expect users to use the oldest available Roslyn version that works for their use case to support as many VS/.NET versions as possible. I would also prefer if we gave the `const`s here more descriptive names, rather than relying on the doc comment.